maven-model-builder/src/main/java/org/apache/maven/model/profile/ProfileActivationContext.java

```diff
@@ -30,6 +30,12 @@
  */
 public interface ProfileActivationContext
 {
+    /**
+     * Key of the property containing the project's packaging.
+     * Available in {@link #getUserProperties()}.
+     * @since 4.0 
+     */
+    String PROPERTY_NAME_PACKAGING = "project.packaging";
```

Comment:
Shouldn't that be `packaging` to align with https://github.com/apache/maven/blob/2a9f39336cec1d8e52d30cc48503d51ed8672536/maven-model-builder/src/main/java/org/apache/maven/model/path/ProfileActivationFilePathInterpolator.java#L75-L82
I think the same reasoning apply, we don't support all `project.*` properties...

Reply:
Good point, will change that